tests/testthat/setup.R

```diff
@@ -0,0 +1,31 @@
+key <- cyphr::key_sodium(
+  sodium::keygen(seed = charToRaw(Sys.getenv("MEETUPR_PWD")))
+)
+
+temptoken <- tempfile(fileext = ".rds")
```

Comment:
as this is a temporary file it wouldn't even end up in the artefacts in case of failure, if I follow things correctly...